src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_controlnet.py

```diff
@@ -671,13 +778,20 @@ def __call__(
             list of `bool`s denoting whether the corresponding generated image likely represents "not-safe-for-work"
             (nsfw) content, according to the `safety_checker`.
         """
+
+        # TODO: add conversion image array to ControlNetConditions
+        if controlnet_conditions is None:
```

Comment:
```suggestion
```

Let's maybe always convert the image to controlnet conditionings to check internally, but wouldn't expose to the user.

Reply:
Fixed in https://github.com/huggingface/diffusers/pull/2627/commits/d1acef4b7c372f9a1c77f526b0d67f727b81064d